network/src/commonMain/kotlin/com/wire/kalium/network/KaliumKtorCustomLogging.kt

```diff
@@ -240,6 +249,49 @@ public class KaliumKtorCustomLogging private constructor(
     }
 }
 
+@Suppress("TooGenericExceptionCaught")
+private fun obfuscateAndLogMessage(text: String): String {
+    return try {
+        val obj = (Json.decodeFromString(text) as JsonElement)
+        if (obj.jsonArray.size > 0) {
+            obj.jsonArray.map {
+                obfuscateJsonElement(it)
+            }
+        } else {
+            obfuscateJsonElement(obj)
+        }
+        obj.toString()
```

Comment:
Why does it return a String if it's ignored?

Reply:
I was using it at first and after that removed the log to the function and forgot to remove the return , will update